pkg/trace/api/otlp.go

```diff
@@ -542,6 +557,9 @@ func (o *OTLPReceiver) convertSpan(rattr map[string]string, lib pcommon.Instrume
 		}
 		return true
 	})
+	if svc := getFirstFromMap(span.Meta, peerServiceDefaults...); svc != "" {
+		setMetaOTLP(span, "_dd.peer.service.source", svc)
+	}
```

Comment:
Shouldn't we set `peer.service` ?

Reply:
I'm wondering about the same - RFC said we need to set `_dd.peer.service.source` for OTLP while https://github.com/DataDog/datadog-agent/pull/16103 sets `peer.service`

Reply:
> Shouldn't we set peer.service ?

We need both `peer.service` and `_dd.peer.service.source`. 

For example, if we choose `db.instance` as the default value for `peer.service`, then we need the following tags set on the span:

```
span['peer.service'] = span['db.instance']
span['_dd.peer.service.source'] = 'db.instance'
```

Reply:
Thanks for the clarification, I have updated to set `peer.service` and `_dd.peer.service.source` with the correct values. PTAL